static/sass/_patterns_carousel.scss

```diff
@@ -4,92 +4,184 @@
-      height: 3rem;
-      margin: {
-        bottom: 0;
-        left: 0;
-        top: -1.5rem;
-      }
-      opacity: 0;
-      overflow: hidden;
-      padding: 0;
-      pointer-events: none;
-      position: absolute;
-      text-indent: -999px;
-      top: 50%;
-      transition: opacity .165s;
-      width: 3rem;
-
-      @media screen and (min-width: $carousel-buttons-breakpoint) {
-        display: block;
+      &:hover {
+        &::after {
+          background-color: $color-x-light;
+          background-image: url('#{$assets-path}c566f22e-fullscreen-grey_16.svg');
+          background-position: 50% 50%;
+          background-repeat: no-repeat;
+          background-size: 12px 12px;
+          border: 1px solid $color-light;
+          border-radius: 2px;
+          content: '';
+          display: block;
+          height: 24px;
+          pointer-events: none;
+          position: absolute;
+          right: 10px;
+          top: 10px;
+          width: 24px;
+        }
       }
     }
+  }
 
-    &:hover .p-carousel__next,
-    &:hover .p-carousel__prev {
-      opacity: 1;
-      pointer-events: all;
+  // p-carousel__next p-carousel__prev
+  // based on vbox-next vbox-prev from _patterns_maas_modal.scss
+  .p-carousel__next,
+  .p-carousel__prev {
+    background: {
+      color: $color-x-light;
+      position: center;
+      repeat: no-repeat;
+      size: .25rem;
+    }
+    border-radius: 3rem;
+    box-sizing: content-box;
+    cursor: pointer;
+    height: 3rem;
+    margin: {
+      bottom: 0;
+      left: 0;
+      right: 0 !important;
+      top: -1.5rem;
     }
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    text-indent: -999px;
+    top: 50%;
+    transition: opacity .165s;
+    width: 3rem;
+    z-index: 10;
 
-    .p-carousel__next:disabled,
-    .p-carousel__prev:disabled {
+    &.swiper-button-disabled {
       cursor: default;
       opacity: 0;
     }
 
-    .p-carousel__prev {
-      background-image: url('#{$assets-path}e8d2e45f-chevron-left.svg');
-      left: -1rem;
+    @media screen and (max-width: $breakpoint-small) {
+      display: none;
+    }
+  }
 
-      @media screen and (min-width: $breakpoint-large) {
-        left: -1.5rem;
-      }
+  &:hover .p-carousel__next,
+  &:hover .p-carousel__prev {
+    pointer-events: all;
+  }
+
+  .p-carousel__prev {
+    background-image: url('#{$assets-path}e8d2e45f-chevron-left.svg');
+    left: -1rem;
+
+    @media screen and (min-width: $breakpoint-large) {
+      left: -1.5rem;
     }
+  }
 
-    .p-carousel__next {
-      background-image: url('#{$assets-path}9716bbab-chevron.svg');
-      right: -1rem;
+  .p-carousel__next {
+    background-image: url('#{$assets-path}9716bbab-chevron.svg');
+    right: -1rem;
 
-      @media screen and (min-width: $breakpoint-large) {
-        right: -1.5rem;
-      }
+    @media screen and (min-width: $breakpoint-large) {
+      right: -1.5rem;
+    }
+  }
 
+  .swiper-container {
```

Comment:
If these are copied from swiper would be good to comment with a link to CSS file on GH or a path in project, so anyone knows where to find it.